sling/nlp/document/text-tokenizer.cc

```diff
@@ -213,14 +213,23 @@ BreakType TokenizerText::BreakLevel(int index) const {
   }
   if (flags & TOKEN_EOS) return SENTENCE_BREAK;
   if (flags & TOKEN_LINE) return LINE_BREAK;
-  if (flags & TOKEN_DISCARD) {
-    // Do not insert space break for zero-width space.
-    return elements_[index].ch == 0x200B ? NO_BREAK : SPACE_BREAK;
-  }
+  if (flags & TOKEN_NONBREAK) return NO_BREAK;
+  if (flags & TOKEN_DISCARD) return SPACE_BREAK;
 
   return NO_BREAK;
 }
 
+int TokenizerText::Style(int index) const {
+  TokenFlags flags = elements_[index].flags;
+  if (flags & TOKEN_TAG) {
+    TokenFlags style = flags & TOKEN_STYLE_MASK;
+    if (style != 0) {
+      return 1 << ((flags & TOKEN_STYLE_MASK) >> TOKEN_STYLE_SHIFT);
```

Comment:
1 << (style >> TOKEN_STYLE_SHIFT) ? (for simplicity)

Reply:
Done. Looks nicer.